### PR TITLE
Feature/jm 6250

### PR DESCRIPTION
**Pull request by annemariesteenkamp**
### Change description ###
Non-sitting days and bank holiday pages and associated pages. Add, and delete non-sitting days 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: server/routes/administration/non-sitting-days/non-sitting-days.controller.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      validatorResult = validate(req.body, validator());
+      if (typeof validatorResult !== 'undefined') {
+        req.session.errors = validatorResult;
+        req.session.formFields = req.body;
+
+        return res.redirect(app.namedRoutes.build('administration.add-non-sitting-days.get'));
+      };
+
+      const payload = {
+        'date': req.body.nonSittingDate !== '' ? dateFilter(req.body.nonSittingDate, null, 'YYYY-MM-DD') : '',
```
Comment (gregs-cgi) [inline server/routes/administration/non-sitting-days/non-sitting-days.controller.js:122]: Should a missing nonSittingDate fail the validation? If it does, this check probably isn't needed.

File: server/routes/administration/non-sitting-days/non-sitting-days.controller.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  };
+
+  module.exports.postNonSittingDays = function(app) {
+    return async function(req, res) {
+
+      try {
+        return res.redirect(app.namedRoutes.build('administration.add-non-sitting-days.get'));
+      } catch (err) {
+
+        app.logger.crit('Failed eo add the non sitting day ', {
```
Comment (gregs-cgi) [inline server/routes/administration/non-sitting-days/non-sitting-days.controller.js:63]: World's most inoffensive typo :D

**Review by gregs-cgi — APPROVED** [review]
Couple of comments but everything looks fine.

File: server/routes/administration/non-sitting-days/non-sitting-days.controller.js (hunk 1 above)
Comment (annemariesteenkamp) [reply server/routes/administration/non-sitting-days/non-sitting-days.controller.js:122]: It is validated, I have removed the check.

**Pull request merged**